project-workflow-trigger-options_5d42dcce042863478675bb99.md

```diff
@@ -45,6 +45,9 @@ When you choose this option Semaphore will create a workflow for every push to t
 In every job from this workflow, Semaphore will export
 `SEMAPHORE_GIT_REF_TYPE` environment variable with value `pull-request`.
 
+Semaphore will use the MERGE commit to run a workflow. The SHA of the HEAD commit 
+of the Pull Request will be stored in `SEMAPHORE\_GIT\_PR\_SHA` environment variable.
```

Comment:
Technical writing is almost exclusively in the simple present tense. Consider rewriting the sentence to remove the future tenses.

Example:

> Semaphore uses the MERGE commit to run a workflow. The SHA of the HEAD commit 
of the Pull Request is stored in `SEMAPHORE\_GIT\_PR\_SHA` environment variable.